totalRP3/core/impl/slash.lua

```diff
@@ -78,6 +99,58 @@ function SlashCmdList.TOTALRP3(msg, editbox)
 	end
 end
 
+local function testConditions(conditionals)
+	logger:Info(Ellyb.Tables.toString(conditionals));
+	local conditionalsPassed = false;
+	for _, condition in pairs(conditionals) do
+		--cleanup the square brackets to give us a comma seperated lsit of tests
+		local passed = true;
+		for test in string.gmatch(condition, "[^, ]+") do
+			if test ~= "" then
+				local args = {strsplit(":", test)};
+				if CONDITIONALS[args[1]] then
+					if not CONDITIONALS[args[1]].test(unpack(args)) then
+						passed = false;
+						break;
+					end
+				elseif SecureCmdOptionParse("["..test.."] true; false")=="false" then
+					passed = false;
+					break;
+				end
+			end
+		end
+		if passed then
+			conditionalsPassed = true
+			break;

```

Comment:
At this point you might as well just return rather than breaking-then-returning? Also means you can kill `conditionalsPassed` entirely and just `return true` here and `return false` at the tail of the function.